.github/workflows/linux.yml

```diff
@@ -0,0 +1,32 @@
+name: Test on Linux
+
+on: push
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
```

Comment:
We're okay with dropping support for node 8. We can change this to `[10.x, 12.x, 14.x]` and that might make this CI pass